Clicker/Supporting/Ratings.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Ratings.swift
+//  Clicker
+//
+//  Created by Lucy Xu on 3/24/19.
+//  Copyright © 2019 CornellAppDev. All rights reserved.
+//
+
+import Foundation
+import StoreKit
+
+class Ratings {
+    
+    static let launchThreshold = 10
```

Comment:
i think it would be better to make this a singleton so that we don't actually instantiate an instance of `Ratings`. so do something like
```
static let shared = Ratings()

private init() {}
```
You would also remove all the `class` declarations before every function and then anywhere you want to call Ratings you can do `Ratings.shared.updateNumAppLaunches()`